examples/small-study/src/background.js

```diff
@@ -0,0 +1,76 @@
+// Allow outside override for debug and testing using prefs
+const STUDYPREFS = {
+  variationPref: `shield.${addonWidgetId}.testingBranch`,
+  firstrunPref: `shield.${addonWidgetId}.firstrun`,
```

Comment:
What's this `firstrunPref` for?

Reply:
Also, seems like we should also have an override pref for study duration and  a method in the API to set up study duration and check for expiration?

Reply:
firstrunPref is 'installTimestamp', and indeed, used for exactly what you describe.